ILL/IDF/d16b_generateIDF.py

```diff
@@ -0,0 +1,125 @@
+import os
+import sys
+from math import asin, pi, sin, cos
+from lxml import etree as le
+import numpy as np
+
+path = os.path.abspath("")
+sys.path.insert(0, path)
+from helper import MantidGeom
+
+# unit is metre
+instrumentName = 'D16B'
+validFrom = "2022-12-01 00:00:00"
+
+monochromator_source = -2.8
+
+# 2 monitors
+zMon1 = -1  # TODO distance ?
```

Comment:
Can we resolve the TODO ? I do not know if the precise distance is useful here, if not, we can keep -1.

Reply:
The Time of flight is not relevant in this case, so an arbitrary distance should be fine. I am going to update the comment.